lib/core/src/main/java/io/atlasmap/core/validate/BaseModuleValidationService.java

```diff
@@ -151,19 +149,15 @@ protected void validateMapMapping(Mapping mapping, List<Validation> validations)
         if (getMode() == AtlasModuleMode.SOURCE) {
             FieldGroup sourceFieldGroup = mapping.getInputFieldGroup();
             if (sourceFieldGroup != null) {
-                multipleFieldSelectionValidator.validate(validations, mappingId,
```

Comment:
Can we skip this only when it's expression mapping?

Reply:
Fair point, but instead of enabling this validation back, I've made it behave so that when you select a collection + any other field then only the first element of a collection is used. You can of course apply concatenate transformation on a collection field and have all elements.

Reply:
Created https://github.com/atlasmap/atlasmap/issues/1690

Reply:
That sounds good! maybe also add a WARN validation result when it only picks up first one? we can file separate issue.

Reply:
Cool, I'll add a warning shortly. No need for a separate issue.

Reply:
Done!